log_extractor.py

```diff
@@ -436,8 +444,45 @@ def parse_logs(self):
                             )
 
 
-# helper.download_artifact(job_url=TEST_URL, dst=TEST_DST)
-log_extractor = LogExtractor(TEST_DST, logs=TEST_LOGS)
-# log-extractor.extract_all(path=TEST_DST)
-log_extractor.parse_art_logs()
-log_extractor.parse_logs()
+@click.command()
+@click.option("--view", help="View where the job is", required=True)
+@click.option("--job", help="Job name", required=True)
+@click.option("--build", help="build number of the job")
+@click.option(
+    "--folder", help="Folder path to save the logs",
+    default=os.path.join(user.home, "art-tests-logs")
+)
+def run(view, job, build, folder):
+    """
+    Extract logs from Jenkins jobs.
+    """
+    # folder = os.path.join(folder, build)
+    if not os.path.exists(path=folder):
+        os.mkdir(folder)
+
+    view_folder = os.path.join(folder, view)
+    if not os.path.exists(path=view_folder):
+        os.mkdir(view_folder)
+
+    job_folder = os.path.join(view_folder, job)
+    if not os.path.exists(path=job_folder):
+        os.mkdir(job_folder)
+
+    build_folder = os.path.join(job_folder, build)
+    if not os.path.exists(path=build_folder):
+        os.mkdir(build_folder)
+
+    job_url = (
```

Comment:
hm do not sure if it good enough for example 4.0 job link looks like https://rhev-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/4.0v4-GE-Tier1-x86_64/49/
or if you go via views
https://rhev-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/4.0/view/GE/job/4.0v4-GE-Tier2-x86_64/49/
I believe it better to send link to the build as parameter and if you want to create dedicated directories
you can split by '/' and take relevant part for the build number and for the job name